docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,228 @@
+.. _ugrid operations:
+
+Working with Mesh Data
+***********************
+
+.. note:: Several of the operations below rely on the optional dependencies
+          mentioned in :doc:`partner_packages`.
+
+..
+    Have a table here that lists the headings below, including a small note
+     about whether it's unchanged (e.g. Saving), has to be done differently
+     (e.g. Extraction), or isn't yet possible (e.g. Arithmetic). Each row
+     should link to the section below. Basically an enhanced TOC.
+
+..
+    Below: use demo code over prose wherever workable. Headings aren't an
+     exhaustive list (can you think of any other popular operations?).
+
+Making a Cube with a Mesh
+-------------------------
+Creating Iris objects from scratch is a highly useful skill for testing code
+and improving understanding of how Iris works. This knowledge will likely prove
+particularly useful when converting data into the Iris mesh data model from
+structured formats and non-UGRID mesh formats.
+
+The objects created in this example will be used where possible in the
+subsequent example operations on this page.
+
+Create a :class:`~iris.experimental.ugrid.Mesh` :
+
+.. doctest:: ugrid_operations
+
+    >>> import numpy as np
+
+    >>> from iris.coords import AuxCoord
+    >>> from iris.experimental.ugrid import Connectivity, Mesh
+
+    # Going to create the following mesh
+    #  (node indices are shown to aid understanding):
+    #
+    #  0----1
+    #  |    |\
+    #  | +  |+\
+    #  2----3--4
```

Comment:
Could you not use the very-similar example in UGRID itself here ?
It's in [this section](https://ugrid-conventions.github.io/ugrid-conventions/#2d-flexible-mesh-mixed-triangles-quadrilaterals-etc-topology) - but you have to scroll down to the CDL example, as they don't provide many anchor points on the page.